zk_toolbox/crates/zk_inception/src/commands/args/run_server.rs

```diff
@@ -17,6 +17,6 @@ pub struct RunServerArgs {
     additional_args: Vec<String>,
     #[clap(long, help = MSG_SERVER_BUILD_HELP)]
     pub build: bool,
-    #[clap(help=MSG_SERVER_URING_HELP, long, default_missing_value = "true", num_args = 0..=1)]
+    #[clap(help=MSG_SERVER_URING_HELP, long, default_missing_value = "true")]
```

Comment:
fyi, the previous version was violating a debug_assert. We only compile with --release in CI, so that's why it was missed earlier, I suppose.